Header header
int32 ts
float32 approx_x
float32 approx_y
float32 approx_z